N/nlohmann_json/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "nlohmann_json"
```

Comment:
Not a fan of this name, but I guess `json` would be incredibly generic.  Is this packaged in any Linux distribution?  If so, how they call it there?

Reply:
That's the package's regular name. See e.g. https://pypi.org/project/nlohmann-json/ or https://packages.ubuntu.com/search?keywords=nlohmann-json3-dev .

Reply:
That's it then.